wheel/pyproject.toml

```diff
@@ -4,3 +4,4 @@ build-backend = "maturin"
 
 [tool.maturin]
 bindings = "pyo3"
+python-source = "python"
```

Comment:
Good job tracking this down and fixing it. This likely happened because we didn't pin a version of `maturin` (and I haven't updated my venv for a while, so didn't see the deprecation warnings). Maybe we consider pinning `maturin` version?

Reply:
yeah, probably. I like pinning things as much as possible and chose the time myself when I need to update CI